PendingReleaseNotes.md

```diff
@@ -11,6 +11,7 @@
 - OSD refactor: drop support for Rook legacy OSD, directory OSD and Filestore OSD. For more details refer to the [corresponding issue](https://github.com/rook/rook/issues/4724).
 - OSD on PVC now supports a metadata device, [refer to the cluster on PVC section](Documentation/ceph-cluster-crd.html#dedicated-metatada-device) or the [corresponding issue](https://github.com/rook/rook/issues/3852).
 - OSD on PVC now supports PVC expansion, if the size of the underlying block increases the Bluestore main block and the overall storage capacity will grow up.
+_ OSD on PVC doesn't use LVM anymore to configure OSD, but solely relies on the entire block device, done [here](https://github.com/rook/rook/pull/4435).
```

Comment:
- nit: `-` instead of `_`
- Raw mode is true for both PVC and non-PVC, right? Should this say something like this? "OSDs are not configured with LVM anymore unless you require encryption or multiple OSDs per device."

Reply:
No raw mode is only true for PVC the non-PVC is tracked here https://github.com/rook/rook/issues/4768.